docs/run/advanced-runtime-options.mdx

```diff
@@ -8,6 +8,10 @@ description: Specify options when building with Qiskit runtime primitives
 
 When calling the primitives, you can pass in options by using the `Options` class or when using the `run` method. In the `Options` class, commonly used options, such as `resilience_level`, are at the first level. Other options are grouped into different categories:
 
+<Admonition type="note" title="Important:">
+To ensure faster and more efficient results, as of 1 Mar 2024, circuits and observables need to be transformed to only use instructions supported by the system (referred to as *instruction set architecture (ISA)* circuits and observables) before being submitted to the Qiskit Runtime primitives.  See the [transpilation documentation](../transpile) for instructions to transform circuits.  Due to this change, the primitives will no longer perform layout or routing operations, consequently transpilation options referring to those tasks will no longer have any effect. Users may still request that the Primitives do no optimization of input circuits via `options.transpilation.skip_transpilation`.
```

Comment:
@beckykd I made the dates consistent in all the admonitions, but I didn't copy the text changes I made here to the other ones. Can you copy this text to the other locations?